DragaliaAPI/Controllers/Dragalia/LoadController.cs

```diff
@@ -66,14 +66,17 @@ public async Task<DragaliaResult> Index()
                 ability_crest_list = savefile.AbilityCrestList.Select(
                     this.mapper.Map<AbilityCrestList>
                 ),
+                dragon_gift_list = savefile.DragonGiftList
+                    .Where(x => x.DragonGiftId > DragonGifts.GoldenChalice)
+                    .Select(this.mapper.Map<DragonGiftList>),
                 talisman_list = savefile.TalismanList.Select(this.mapper.Map<TalismanList>),
                 weapon_body_list = savefile.WeaponBodyList.Select(this.mapper.Map<WeaponBodyList>),
                 party_list = savefile.PartyList.Select(this.mapper.Map<PartyList>),
                 quest_story_list = savefile.StoryStates
                     .Where(x => x.StoryType == StoryTypes.Quest)
                     .Select(mapper.Map<QuestStoryList>),
                 unit_story_list = savefile.StoryStates
-                    .Where(x => x.StoryType == StoryTypes.Chara)
+                    .Where(x => x.StoryType == StoryTypes.Chara || x.StoryType == StoryTypes.Dragon)
```

Comment:
dragon isn't an actual story type, both chara and dragon are all stored in unit_story_list so we should probably rework the enum